lib/base-controller.js

```diff
@@ -47,15 +50,14 @@ module.exports = class BaseController extends Controller {
       };
 
       return evalCondition(value.condition) ?
-        req.baseUrl + value.target :
+        normaliseUrl(value.target) :
         result;
     }, next);
-
     if ((req.params.action === 'edit') && completed(next)) {
       // The user is editing the form and has already completed the next
       // step, so let's check whether we should fast-forward them to the
       // confirm page
-      next = (!this.options.continueOnEdit || next === confirmStep) ?
+      next = (!req.form.options.continueOnEdit || next === confirmStep) ?
         confirmStep :
         next + '/edit';
     }
```

Comment:
This logic exists in hof-form-controller, not sure why its still here too, currently it looks like it is performing forking logic twice